pkg/scheduler/pod.go

```diff
@@ -26,18 +27,20 @@ func IsCompletedPod(pod *v1.Pod) bool {
 }
 
 func IsGPUPod(pod *v1.Pod) bool {
-	if GetResourceRequests(pod, v1alpha1.ResourceGPUCore) > 0 || GetResourceRequests(pod, v1alpha1.ResourceGPUMemory) > 0 {
-		return true
-	}
-
-	if GetResourceRequests(pod, v1alpha1.ResourceQGPUCore) > 0 || GetResourceRequests(pod, v1alpha1.ResourceQGPUMemory) > 0 {
+	if IsResourceExists(pod, v1alpha1.ResourceGPUCore) || IsResourceExists(pod, v1alpha1.ResourceGPUMemory) ||
```

Comment:
Still leave `GetResourceRequests` as it is for we may need it days later. If you feel like to remove it, I'm ok with it.